examples/beam/src/main/java/edu/snu/nemo/examples/beam/AlternatingLeastSquare.java

```diff
@@ -382,6 +395,19 @@ public void processElement(final ProcessContext c) throws Exception {
       itemMatrix = itemMatrix.apply(new UpdateUserAndItemMatrix(numFeatures, lambda, parsedUserData, parsedItemData));
     }
 
+    final PCollection<String> result = itemMatrix.apply(MapElements.<KV<Integer, List<Double>>, String>via(
```

Comment:
if (checkOutput) {
  result = itemMatrix.apply(...);
  write(result);
}

so that we don't execute `itemMatrix.apply(...);` when not writing the output.